instrumentation/nginx/src/script.h

```diff
@@ -26,6 +26,10 @@ struct NgxCompiledScript {
 
     return true;
   }
+
+  bool isValid() {
```

Comment:
Perhaps nitpicking, but maybe `isEmpty`? `isValid` has more of a "is in an error state" vibe :smile:

Reply:
Yes, `IsEmpty` sounds better